workflow/fixtures/workflow_data.json

```diff
@@ -0,0 +1 @@
8, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 359, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 360, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 361, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 362, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 363, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 364, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 365, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 366, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 367, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 368, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 369, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 370, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 371, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 372, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 373, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 374, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 375, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 376, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 377, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 378, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 379, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 380, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 381, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 382, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 383, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 384, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 385, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 386, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 387, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 388, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 389, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 390, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 391, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 392, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 393, "fields": {"workflow": 2}}, {"model": "workflow.case", "pk": 394, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 395, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 396, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 397, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 398, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 399, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 400, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 401, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 402, "fields": {"workflow": 1}}, {"model": "workflow.case", "pk": 403, "fields": {"workflow": 1}}]
```

Comment:
This fixture is kinda messy. Lets make a ticket to make a clean one

Reply:
Sounds good. I think it's worth having this in develop for now though

Reply:
Agreed